scene/2d/canvas_item.h

```diff
@@ -229,6 +229,7 @@ class CanvasItem : public Node {
 	// Used to resize/move/select the node
 	virtual void _edit_set_rect(const Rect2 &p_rect){};
 	virtual Rect2 _edit_get_rect() const { return Rect2(-32, -32, 64, 64); };
+	virtual bool _edit_is_clicked(const Point2 &p_point, double p_tolerance) const { return true; }
```

Comment:
`_edit_is_clicked` is a little bit confusing I think, as it makes me think the function is going to return whether the object is actually clicked or not. (thus depending on the actual state of the mouse).
Maybe we could go for something a little bit like "_edit_is_selected_on_click", so it is more obvious what the function does. (It is more understandable in this case that the object is not clicked yet)

Reply:
Changed.